pyproject.toml

```diff
@@ -0,0 +1,17 @@
+[tool.isort]
+profile = "black"
+sections = [
+  "FUTURE",
+  "STDLIB",
+  "THIRDPARTY",
+  "ARCHSPEC", "LLNL", "FIRSTPARTY",
+  "LOCALFOLDER",
+]
+known_first_party = "spack"
+known_archspec = "archspec"
+known_llnl = "llnl"
+src_paths = "lib"
+force_sort_within_sections = false
+line_length = 79
+honor_noqa = true
+use_parentheses = true
```

Comment:
This can be removed, it's implied by `profile = 'black'`

Reply:
isort also has a `color_output` option we could use.

Reply:
done